proposals/dao/timelock_admin.ts

```diff
@@ -0,0 +1,52 @@
+import { ERC20PCVDepositWrapper__factory } from '@custom-types/contracts';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { expect } from 'chai';
+import hre from 'hardhat';
+
+const fipNumber = 'timelock_admin';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const factory = (await hre.ethers.getContractFactory('ERC20PCVDepositWrapper')) as ERC20PCVDepositWrapper__factory;
+  const rariTimelockFeiOldLens = await factory.deploy(addresses.rariInfraFeiTimelock, addresses.fei, true);
```

Comment:
Will make sure to add lenses for contracts with significant Fei that are becoming part of the Fei ecosystem in the future!